### DISCUSSION TITLE
Dependabot support for TFLint

### DISCUSSION THREAD

**Discussion by stevehipwell** [terraform-linters/tflint#1907] [Ideas]
I'd like Dependabot to be able to open PRs to update my TFLint plugin versions.

**Comment by PatMyron** [comment]
linking the dependabot issue you opened:
https://github.com/dependabot/dependabot-core/issues/7515

---

Also linking a Dependabot competitor's `tflint` support:
https://docs.renovatebot.com/modules/manager/tflint-plugin/
